checks/exec.go

```diff
@@ -73,8 +151,46 @@ func runCmd(cmd *osExec.Cmd, result *pkg.CheckResult) (results pkg.Results) {
 	}
 	result.AddDetails(details)
 	if details.ExitCode != 0 {
-		return result.Failf("non-zero exit-code: %d: %s %s", details.ExitCode, details.Stdout, details.Stderr).ToSlice()
+		return result.Failf("non-zero exit-code: %d. (stdout=%s) (stderr=%s)", details.ExitCode, details.Stdout, details.Stderr).ToSlice()
 	}
+
 	results = append(results, result)
 	return results
 }
+
+func saveConfig(dirPrefix string, configTemplate *textTemplate.Template, view any) (string, error) {
+	dirPath, err := os.MkdirTemp("", dirPrefix)
```

Comment:
The credentials should not be in the /tmp folder which is world readable, do it in the current directory